### PR TITLE
TheIridiumAuthors.IridiumBrowser version 2020.11.85

### PR DESCRIPTION
**Pull request by DeerMaximum**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/15097)

### CODE REVIEW

**Comment by denelon** [comment]
@ItzLevvie I don't see Iridium in the repository, are you addressing this package?

File: manifests/t/TheIridiumAuthors/IridiumBrowser/2020.11.85/TheIridiumAuthors.IridiumBrowser.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+PackageIdentifier: TheIridiumAuthors.IridiumBrowser
+PackageVersion: 2020.11.85
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://dl.iridiumbrowser.de/windows/2020.11.85/iridiumbrowser-2020.11.85.0-x64.msi
+  InstallerSha256: 81B2EB50351BE0A441EA25F45434FC11D0ED59B7B4D28FF9F08A18B08DB09F6B
+  ProductCode: '{BDA0D317-FDBB-46FE-BEB4-2852C91B82B1}'
```
Comment (ItzLevvie) [inline manifests/t/TheIridiumAuthors/IridiumBrowser/2020.11.85/TheIridiumAuthors.IridiumBrowser.installer.yaml:11]:
```suggestion
  ProductCode: '{BDA0D317-FDBB-46FE-BEB4-2852C91B82B1}'
- Architecture: x86
  InstallerType: msi
  InstallerUrl: https://dl.iridiumbrowser.de/windows/2020.11.85/iridiumbrowser-2020.11.85.0-x86.msi
  InstallerSha256: 8BFE6F24FF0717C6850701BE59F7E8083281A7F73D085075BA4E4081ED21B0F6
  ProductCode: '{28BE43B3-C34C-4F76-A9B0-BD8C4965F922}'
```

File: manifests/t/TheIridiumAuthors/IridiumBrowser/2020.11.85/TheIridiumAuthors.IridiumBrowser.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: TheIridiumAuthors.IridiumBrowser
+PackageVersion: 2020.11.85
+Installers:
+- InstallerLocale: en-US
```
Comment (ItzLevvie) [inline manifests/t/TheIridiumAuthors/IridiumBrowser/2020.11.85/TheIridiumAuthors.IridiumBrowser.installer.yaml:6]:
```suggestion
```

File: manifests/t/TheIridiumAuthors/IridiumBrowser/2020.11.85/TheIridiumAuthors.IridiumBrowser.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: TheIridiumAuthors.IridiumBrowser
+PackageVersion: 2020.11.85
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
```
Comment (ItzLevvie) [inline manifests/t/TheIridiumAuthors/IridiumBrowser/2020.11.85/TheIridiumAuthors.IridiumBrowser.installer.yaml:7]:
```suggestion
- Architecture: x64
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/t/TheIridiumAuthors/IridiumBrowser/2020.11.85/TheIridiumAuthors.IridiumBrowser.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: TheIridiumAuthors.IridiumBrowser
+PackageVersion: 2020.11.85
```
Comment (ItzLevvie) [inline manifests/t/TheIridiumAuthors/IridiumBrowser/2020.11.85/TheIridiumAuthors.IridiumBrowser.installer.yaml:4]:
```suggestion
PackageVersion: 85.0.0
```

File: manifests/t/TheIridiumAuthors/IridiumBrowser/2020.11.85/TheIridiumAuthors.IridiumBrowser.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: TheIridiumAuthors.IridiumBrowser
+PackageVersion: 2020.11.85
```
Comment (ItzLevvie) [inline manifests/t/TheIridiumAuthors/IridiumBrowser/2020.11.85/TheIridiumAuthors.IridiumBrowser.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 85.0.0
```

File: manifests/t/TheIridiumAuthors/IridiumBrowser/2020.11.85/TheIridiumAuthors.IridiumBrowser.yaml [hunk 6]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: TheIridiumAuthors.IridiumBrowser
+PackageVersion: 2020.11.85
```
Comment (ItzLevvie) [inline manifests/t/TheIridiumAuthors/IridiumBrowser/2020.11.85/TheIridiumAuthors.IridiumBrowser.yaml:4]:
```suggestion
PackageVersion: 85.0.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Pull request closed without merging**